src/screens/ContactScreen/elements/ItemUser.js

```diff
@@ -34,7 +36,7 @@ const ItemUser = ({photo, username, followed, onPress, userid}) => {
         style={[
           styles.containerCard,
           {
-            backgroundColor: isSelect ? COLORS.holytosca15percent : COLORS.white
+            backgroundColor: isSelect ? anonColorCondition : COLORS.white
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

This change correctly applies the `anonColorCondition` to the background color when `isSelect` is true. Good use of conditional rendering based on the `isAnon` prop.
```diff
- backgroundColor: isSelect ? COLORS.holytosca15percent : COLORS.white
+ backgroundColor: isSelect ? anonColorCondition : COLORS.white
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->